apps/api/package.json

```diff
@@ -86,6 +86,7 @@
     "dotenv": "^8.2.0",
     "envalid": "^6.0.1",
     "handlebars": "^4.7.7",
+    "handlebars-dateformat": "^1.1.3",
```

Comment:
Hi @chavda-bhavik thanks for adding this, I was looking a bit deeper in to this package and it seems it's a 1 line of code using momentjs. Internally we decided to switch from moment to `date-fns` because it was deprecated by it's maintainer. 

I would suggest implementing our own date formatted using `date-fns`, and perhaps crediting the amazing work of https://github.com/tcort/handlebars-dateformat/blob/master/index.js on this library, and even if you are up to it contributing it upstream to his packages if he is in to migrating from moment js.